PlaybackLab/SerializationDownloadsSampleApp/SerializationDownloadsSampleApp/Views/AssetTableViewController.swift

```diff
@@ -0,0 +1,119 @@
+//
+//  AssetTableViewController.swift
+//  SerializationDownloadsSampleApp
+//
+//  Copyright © 2018 Ooyala. All rights reserved.
+//
+
+import UIKit
+
+let PLAYER_SEGUE  = "PlayerViewControllerSegue"
+
+class AssetTableViewController: UITableViewController, OptionTableViewCellDelegate {
+  
+  private let options = OptionDataSource.options()
+  
+  override func viewDidLoad() {
+    super.viewDidLoad()
+    tableView.tableFooterView = UIView()
+    tableView.bounces = false
+    tableView.rowHeight = 75.0
+    
+  }
+  
+  override var supportedInterfaceOrientations: UIInterfaceOrientationMask {
+    return .portrait
+  }
+  
+  // MARK: - Table view data source
+  override func tableView(_ tableView: UITableView, numberOfRowsInSection section: Int) -> Int {
+    return options.count
+  }
+  
+  override func tableView(_ tableView: UITableView, cellForRowAt indexPath: IndexPath) -> UITableViewCell {
+    let cell = tableView.dequeueReusableCell(withIdentifier: CELL_REUSE_IDENTIFIER, for: indexPath) as! OptionTableViewCell
+    let option = options[indexPath.row] 
+    cell.setup(delegate: self, option: option)
+    return cell
+  }
+  
+  override func tableView(_ tableView: UITableView, accessoryButtonTappedForRowWith indexPath: IndexPath) {
+    let cell = tableView.cellForRow(at: indexPath) as! OptionTableViewCell
+    let option = cell.option as! PlayerSelectionOption
```

Comment:
this line is throwing a warning error because of the cast. I don't think you need the `as!` since it already knows its a `PlayerSelectionOption`. The point here is can you fix the warning?

Reply:
Done, the warning was removed it.